components/RevisionCard.vue

```diff
@@ -175,6 +175,62 @@
           <span class="sr-only">{{$t(`Label-Loading`)}}...</span>
         </div>
       </div>
+      <div v-if="enableCesp">
+        <div v-if="displayChoiceAuthor && (choiceInfoAuthor.type == 'warning')" v-bind:style="{color: 'red', 'margin-left': '50px', 'margin-right': '50px'}">
+          WikiLoop-DoubleCheck has detected suspicious behavior by the author of this revision. Recent revisions by this author have an average ORES damaging score of {{choiceInfoAuthor.percentage}}%. Should a warning be sent on your behalf to the author? <br>
+          Previous revisions by this author:
+        </div>
+        <div v-if="displayChoiceAuthor && (choiceInfoAuthor.type == 'block')" v-bind:style="{color: 'red', 'margin-left': '50px', 'margin-right': '50px'}">
```

Comment:
You don't have to pass all individual field, you could just pass in `someItem`, and then call `someItem.displayChoiceAuthor`